apiserver/Makefile

```
@@ -0,0 +1,195 @@
+# entire set of makefiles included in this invocation, looking for lines of the
+# file as xyz: ## something, and then pretty-format the target and help. Then,
+# if there's a line with ##@ something, that gets pretty-printed as a category.
+# More info on the usage of ANSI control characters for terminal formatting:
+# https://en.wikipedia.org/wiki/ANSI_escape_code#SGR_parameters
+# More info on the awk command:
+# http://linuxcommand.org/lc3_adv_awk.php
+
+help: ## Display this help.
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-20s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+##@ Development
+
+fmt: ## Run go fmt against code.
+	go fmt ./...
+
+vet: ## Run go vet against code.
+	go vet ./...
+
+fumpt: gofumpt ## Run gofmtumpt against code.
+	$(GOFUMPT) -l -w .
+
+imports: goimports ## Run goimports against code.
+	$(GOIMPORTS) -l -w .	
+
+test: fmt vet fumpt imports lint  ## Run unit tests.
+	go test ./... -race -coverprofile ray-kube-api-server-coverage.out
+
+lint: fmt vet fumpt imports ## Run the linter.
+	$(GOLANGCI_LINT) run  --timeout=3m	
+
+##@ Build
+
+COMMIT := $(shell git rev-parse --short HEAD)
+VERSION := $(shell git describe --tags $(shell git rev-list --tags --max-count=1))
+DATE=$(shell date -u +'%Y-%m-%dT%H:%M:%SZ')
+REPO=github.com/ray-project/kuberay
+
+BUILD_FLAGS ?= -ldflags="-X '${REPO}/apiserver/cmd/version.Version=$(VERSION)' \
+	-X '${REPO}/apiserver/cmd/version.gitCommit=$(COMMIT)' \
+	-X '${REPO}/apiserver/cmd/version.buildDate=$(DATE)'"
+
+build: fmt vet fumpt imports lint ## Build api server binary.
+	go build $(BUILD_FLAGS) -o ${REPO_ROOT_BIN}/kuberay-apiserver cmd/main.go
+
+run: fmt vet fumpt imports lint ## Run the api server from your host.
+	go run -race cmd/main.go
+
+docker-image: test ## Build image with the api server.
+	${ENGINE} build -t ${IMG} -f Dockerfile ..
+
+docker-push: ## Push image with the api server.
+	${ENGINE} push ${IMG}
+
+##@ Deployment
+.PHONY: install
+install: kustomize ## Install the kuberay api server to the K8s cluster specified in ~/.kube/config.
+	cd deploy/base && $(KUSTOMIZE) edit set image kuberay/apiserver=$(IMG)
+	$(KUSTOMIZE) build deploy/base | kubectl replace -f -
+
+.PHONY: uninstall
+uninstall: ## Install the kuberay api server to the K8s cluster specified in ~/.kube/config.
+	$(KUSTOMIZE) build deploy/base | kubectl delete -f -
+
+.PHONY: deploy
+deploy: ## Deploy via helm the kuberay api server to the K8s cluster specified in ~/.kube/config.
+	helm upgrade --install kuberay-apiserver ../helm-chart/kuberay-apiserver --wait \
+	--set image.tag=${IMG_TAG} --set image.pullPolicy=IfNotPresent
+
+.PHONY: undeploy
+undeploy: ## Undeploy via helm the kuberay api server to the K8s cluster specified in ~/.kube/config.
+	helm delete -n kuberay-apiserver kuberay-apiserver --wait
```

`make deploy` will install the chart in the `default` namespace.

```suggestion
	helm delete kuberay-apiserver --wait
```

---

Thanks, I will fix it.